docs/user-guides/shopping-lists.md

```
@@ -0,0 +1,202 @@
+# Shopping Lists User Guide
+
+**Epicourier-Web v1.3.0**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)